File: javascript/7_kyu/highest_and_lowest.js
// Highest and Lowest
// https://www.codewars.com/kata/554b4ac871d6813a03000035/train/javascript

function highAndLow(numbers){
    var highAndLow = numbers.split(' ').map(Number);
    return [Math.max(...highAndLow).toString(), Math.min(...highAndLow).toString()].join(' ');
}

console.log(highAndLow("4 5 29 54 4 0 -214 542 -64 1 -3 6 -6"));

File: python/7_kyu/highest_and_lowest.py
# Highest and Lowest
# https://www.codewars.com/kata/554b4ac871d6813a03000035/train/python

def high_and_low(numbers):
    return ' '.join([str(max(map(int, numbers.split(' ')))), str(min(map(int, numbers.split( ' '))))])

print(high_and_low("4 5 29 54 4 0 -214 542 -64 1 -3 6 -6"))